recipes-kernel/linux/linux-mainline/arm/defconfig

```diff
@@ -2380,10 +3134,13 @@ CONFIG_FB_TILEBLITTING=y
 # CONFIG_FB_IBM_GXT4500 is not set
 # CONFIG_FB_VIRTUAL is not set
 # CONFIG_FB_METRONOME is not set
-# CONFIG_FB_BROADSHEET is not set
 CONFIG_FB_SIMPLE=y
 # CONFIG_FB_SSD1307 is not set
-CONFIG_BACKLIGHT_LCD_SUPPORT=y
```

Comment:
No Backlight LCD support ??

Reply:
I have Backlight on LCD - and I can justify the brightness in the same way like before. What else do you expect?

The only thing is the wrong adjustment of brightness an AM070-DS2 - but this I found on old warrior branch too!

Reply:
ok